src/theme.ts

```diff
@@ -269,6 +269,24 @@ export async function setTheme(theme?: string): Promise<void> {
     const styleSheet = styleElements.get(stylesheetName)!;
     styleSheet.disabled = false;
 
+    /**
+     * Adds the Compound theme class to the top-most element in the document
+     * This will automatically refresh the colour scales based on the OS or user
+     * preferences
+     *
+     * Note: Theming through Compound is not yet established. Brand theming should
+     * be done in a similar manner as it used to be done.
+     */
+    document.body.classList.remove("cpd-theme-light", "cpd-theme-dark", "cpd-theme-light-hc", "cpd-theme-dark-hc");
+
+    let compoundThemeClassName = `cpd-theme-` + (stylesheetName.includes("light") ? "light" : "dark");
+    // Always respect user OS preference!
+    if (isHighContrastTheme(theme) || window.matchMedia("(prefers-contrast: more)").matches) {
+        compoundThemeClassName += "-hc";
+    }
```

Comment:
I think it would be a good idea for the compound package to export the supported themes indeed.
This is not currently done, but sounds like a worthwhile thing to do at some point.

Reply:
Could you open an issue if you feel like it should be done at a later stage?

Reply:
https://github.com/vector-im/compound/issues/139